### ISSUE TITLE
+ versions not handled

### ISSUE THREAD

**Issue by Sciss** [rtimush/sbt-updates#9]
First of all, cool plugin! I just tried it out. I have the habit to use `+` for the minor (binary compatible) versions; this seems to be ignored by the plugin. For example:

```
[info] Found 4 dependency updates for machinaecoelestis
[info]   de.sciss:mellite             : 0.5.+               -> 0.5.0
[info]   de.sciss:pdflitz             : 1.0.+               -> 1.0.1
[info]   de.sciss:play-json-sealed    : 0.1.+               -> 0.1.0
[info]   org.scala-lang:scala-library : 2.10.3 -> 2.11.0-M5         
```

Obviously `0.5.+` is already using `0.5.0`, also `1.0.+` -> `1.0.1` is debatable. Finally, is there a switch to disable RC and milestone releases, like the last item?

**Comment by jedesah** [comment]
👍

**Comment by omervk** [comment]
+1 to this issue 👍 
Hope it leaves the icebox someday...

**Comment by kiranbayram** [comment]
I've started to give this a shot but I can't move it in project board. Could you please do that for me @rtimush ?

**Comment by kiranbayram** [comment]
Opened https://github.com/rtimush/sbt-updates/pull/104 for this

**Comment by rtimush** [comment]
Fixed in #104
